src/clj/rems/db/form.clj

```diff
@@ -57,3 +57,13 @@
       (create-form-item! user-id form-id index item))
     {:success (not (nil? form-id))
      :id form-id}))
+
+(defn update-form! [command]
+  (let [catalogue-items (db/get-catalogue-items {:form (:id command)})]
```

Comment:
By default `db/get-catalogue-items` doesn't return archived catalogue items. I wonder if that would be a problem here? To get all catalogue items, the `:archived true` parameter is needed. I wonder whether `db/get-catalogue-items` should default to returning all items and hiding the archived should be handled at a higher level (near the api).

Reply:
Hmm interesting, the negative test passes anyway...

Reply:
I guess this works correctly now. An explicit `:archived false` would improve readability.